### PR TITLE
password and api key reset functions

### PR DESCRIPTION
**Pull request by leohemsted**
i've already used these locally to great effect to reset the api keys for the tools account

see https://developer-portal.driver-vehicle-licensing.api.gov.uk/apis/authentication-api/authentication-api-description.html for details - the json version there is a bit more complete too

- [ ] https://github.com/alphagov/notifications-utils/pull/1019

### CODE REVIEW

File: app/clients/letter/dvla.py
Comment (samuelhwilliams) [inline app/clients/letter/dvla.py:122]: 👀

File: app/clients/letter/dvla.py
Comment (samuelhwilliams) [inline app/clients/letter/dvla.py:85]: How do you feel about `change` instead of `reset`?

**Review by samuelhwilliams — COMMENTED** [review]
Looks good. We should probably also tackle the locking strategy in this PR to block off any race conditions.

File: app/clients/letter/dvla.py
Comment (leohemsted) [reply app/clients/letter/dvla.py:122]: whoops fat-thumbed it. will squash out to say just 30 and not spread fake news about.

File: app/clients/letter/dvla.py
Comment (leohemsted) [reply app/clients/letter/dvla.py:85]: yeah i think that's reasonable. do we think we should change that for the password function as well? No strong opinion but I'm leaning to yes - "reset password" often refers to where you don't know an old password. Change also matches the relevant sections in the DVLA api docs for both the password and api key methods

Comment (samuelhwilliams) [reply app/clients/letter/dvla.py:85]: yeah I think both too 👍

Comment (leohemsted) [reply app/clients/letter/dvla.py]: squashed out 🦶

File: requirements.in
Comment (samuelhwilliams) [inline requirements.in:31]: 👀

**Review by samuelhwilliams — COMMENTED** [review]
Looks good on the whole - will wait until the utils PR is merged to give a final 👀  and 👍

**Review by samuelhwilliams — APPROVED** [review]
👍 

This looks good to me. Tested locally and all working as expected.

**Pull request merged**
